server/index.ts

```diff
@@ -104,91 +104,82 @@ app.get('/api/oauth/return', async (request, response, next) => {
   }
 });
 
-app.get('/api/search/artist/:searchq', async (request, response, next) => {
-  const searchQuery = request.params.searchq;
+app.get(
+  '/api/search/:searchcategory/:searchq',
+  async (request, response, next) => {
+    const searchQuery = request.params.searchq;
```

Comment:
would be nicer if you called it 'request.params.searchquery' like with searchcategory. 
waaay more legible :)